agent/probe/process.go

```diff
@@ -2670,37 +2647,31 @@ func (p *Probe) addProcHistory(id string, proc *procInternal, bFromMonitor bool)
 		defer p.unlockProcMux()
 	}
 
-	if histProc, ok = p.procHistoryMap[id]; !ok {
-		histProc = ringbuffer.New(400) // TODO: more ?
-		p.procHistoryMap[id] = histProc
+	histProc, ok := p.procHistoryMap[id]
+	if !ok {
+		histProc = make([]*procInternal, 0)
 	}
 
-	histProc.Write(proc)
-
-	// Verification section, test only
-	// log.WithFields(log.Fields{"path": proc.path, "id": id, "cnt": histProc.Length()}).Debug("PROC: ")
-	// if histProc.Length()() == 400 {
-	//		var list []*procInternal
-	//		elements := histProc.DumpExt()
-	//		for i := 0; i < len(elements); i++ {
-	//			list = append(list, elements[i].(*procInternal))
-	//		}
-	//		printLastProcElements(list, 10)
-	//	}
+	// cap the history length to 400 entries
+	if len(histProc) < 400 {
```

Comment:
Better to make the `400` as a const.

Reply:
It is optional.

Reply:
Agree with @holyspectral here. This is a magic number and should be a constant at least.

Reply:
Done